src/comm.F

```diff
@@ -13747,10 +13746,11 @@ end subroutine comm_all_s_GPU
 !cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
 
 
+#if 0
```

Comment:
if the following subroutine is not used, I would suggest deleting it instead of using #if 0